src/pages/EntityDetails/Model/Model.js

```diff
@@ -139,6 +140,147 @@ const Model = () => {
   };
 
   const LocalAppChips = renderCounts("localApps", modelStatusData);
+  const offersChips = renderCounts("offers", modelStatusData);
+  const remoteAppChips = renderCounts("remoteApps", modelStatusData);
+
+  const localAppTableLength = localApplicationTableRows?.length;
+  const appOffersTableLength = appOffersRows?.length;
+  const remoteAppsTableLength = remoteApplicationTableRows?.length;
+
+  const OffersHeader = () => (
+    <>
+      <span>
+        {appOffersTableLength} {pluralize(appOffersTableLength, "Offer")}
+      </span>
+      <ChipGroup chips={offersChips} descriptor={null} />
+    </>
+  );
+
+  const LocalAppsHeader = () => (
+    <>
+      <span>
+        {localAppTableLength}{" "}
+        {pluralize(localAppTableLength, "Local application")}
+      </span>
+      <ChipGroup chips={LocalAppChips} descriptor={null} />
+    </>
+  );
+
+  const RemoteAppsHeader = () => (
+    <>
+      <span>
+        {remoteAppsTableLength}{" "}
+        {pluralize(remoteAppsTableLength, "Remote application")}
+      </span>
+      <ChipGroup chips={remoteAppChips} descriptor={null} />
+    </>
+  );
+
+  const LocalAppsTable = () => (
+    <>
+      {localAppTableLength > 0 && (
+        <MainTable
+          headers={localApplicationTableHeaders}
+          rows={localApplicationTableRows}
+          className="entity-details__apps p-main-table"
+          sortable
+          emptyStateMsg={"There are no local applications in this model"}
+        />
+      )}
+    </>
+  );
+
+  const AppOffersTable = () => (
+    <>
+      {appOffersTableLength > 0 && (
+        <>
+          <MainTable
+            headers={appsOffersTableHeaders}
+            rows={appOffersRows}
+            className="entity-details__offers p-main-table"
+            sortable
+            emptyStateMsg={"There are no offers associated with this model"}
+          />
+        </>
+      )}
+    </>
+  );
+
+  const RemoteAppsTable = () => (
+    <>
+      {remoteAppsTableLength > 0 && (
+        <>
+          <ChipGroup chips={remoteAppChips} descriptor={null} />
+          <MainTable
+            headers={remoteApplicationTableHeaders}
+            rows={remoteApplicationTableRows}
+            className="entity-details__remote-apps p-main-table"
+            sortable
+            emptyStateMsg={"There are no remote applications in this model"}
+          />
+        </>
+      )}
+    </>
+  );
+
+  const expandedKey = () => {
+    if (appOffersTableLength) {
+      return "offers";
+    }
+    if (localAppTableLength) {
+      return "local-apps";
+    }
+    if (remoteAppsTableLength) {
+      return "remote-apps";
+    }
+  };
+
+  const getContentReveals = () => {
+    return (
+      <>
+        {appOffersTableLength > 0 && (
+          <ContentReveal
+            title={OffersHeader()}
+            showContent={expandedKey() === "offers"}
```

Comment:
The content should always be visible by default and the user can close it if they choose.